.fixtures.yml

```diff
@@ -20,5 +20,3 @@ fixtures:
     facts: 'git://github.com/puppetlabs/puppetlabs-facts.git'
     puppet_agent: 'git://github.com/puppetlabs/puppetlabs-puppet_agent.git'
     provision: 'git://github.com/puppetlabs/provision.git'
-  symlinks:
```

Comment:
@RandomNoun7 Is this OK? Wasn't sure why this was added.

Reply:
As long as it creates the proper symlink in `spec/fixtures/modules` when `spec_prep` is called, sure. It was probably required at one point and maybe is no longer.

Reply:
Yeah, things were working OK without it so looks like it is no longer needed.